package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@a-ui/acpaas": "^4.1.0",
     "@a-ui/core": "^4.1.0",
+    "@a-ui/flexboxgrid": "^2.0.0",
```

Comment:
This can be removed again. If you want you can adjust the CDN import, but just know that there were some breaking changes in this `v2.0.0` version (compared to `v1.0.0`).